pandas/_libs/tslib.pyx

```diff
@@ -266,20 +266,16 @@ def format_array_from_datetime(ndarray[int64_t] values, object tz=None,
         elif basic_format:
 
             dt64_to_dtstruct(val, &dts)
-            res = '%d-%.2d-%.2d %.2d:%.2d:%.2d' % (dts.year,
-                                                   dts.month,
-                                                   dts.day,
-                                                   dts.hour,
-                                                   dts.min,
-                                                   dts.sec)
+            res = (f'{dts.year}-{dts.month}-{dts.day} '
+                   f'{dts.hour}-{dts.min}-{dts.sec}')
```

Comment:
it looks like this is failing.  i suggest reverting this for now to get the tests passing and figure this out in a separate PR

Reply:
I think just need zero padding with digits, so `f"{dts.year}-{dts.month:02}..."`

Reply:
Will open a separate PR, and will try what @WillAyd suggested